worker.go

```diff
@@ -16,21 +26,29 @@ type Logger func(format string, v ...interface{})
 // and a earliest next job reservation time (only if error is present).
 type WorkFunc func(job *Job) (reserveAfter *time.Time, err error)
 
+// JobQueue just do jobs from queue
```

Comment:
I'm not sure if this explanation is not clear or just wrong. It doesn't _do_ jobs, right?
In any case, `.` should be added after first sentence, that's `godoc` convention.